pkg/controllers/vdb/httpservercertgen_reconciler.go

```diff
@@ -59,15 +59,15 @@ func (h *HTTPServerCertGenReconciler) Reconcile(ctx context.Context, _ *ctrl.Req
 	}
 	// If the secret name is set, check that it exists. As a convenience we will
 	// regenerate the secret using the same name.
-	if h.Vdb.Spec.NmaTLSSecret != "" {
-		nm := names.GenNamespacedName(h.Vdb, h.Vdb.Spec.NmaTLSSecret)
+	if h.Vdb.Spec.NMATLSSecret != "" {
+		nm := names.GenNamespacedName(h.Vdb, h.Vdb.Spec.NMATLSSecret)
 		secret := corev1.Secret{}
 		err := h.VRec.Client.Get(ctx, nm, &secret)
 		if errors.IsNotFound(err) {
 			h.Log.Info("httpServerTLSSecret is set but doesn't exist. Will recreate the secret.", "name", nm)
 		} else if err != nil {
 			return ctrl.Result{},
-				fmt.Errorf("failed while attempting to reade the tls secret %s: %w", h.Vdb.Spec.NmaTLSSecret, err)
+				fmt.Errorf("failed while attempting to reade the tls secret %s: %w", h.Vdb.Spec.NMATLSSecret, err)
```

Comment:
Its not your fault, but could you correct the spelling?
```suggestion
				fmt.Errorf("failed while attempting to read the tls secret %s: %w", h.Vdb.Spec.NMATLSSecret, err)
```